build/IntegrationCIBuild.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
```

Comment:
We don't need this.

Reply:
This PR is just bringing over what's in the `feature/integration-tests` branch. I'll make another PR once this merges that includes what's currently in #5111.